web/src/components/storage/PartitionPage.tsx

```diff
@@ -491,10 +493,17 @@ type PartitionDescriptionProps = {
 };
 
 function PartitionDescription({ partition }: PartitionDescriptionProps): React.ReactNode {
+  const label = partition.filesystem?.label;
+
   return (
     <Split hasGutter>
-      <span>{partition.description}</span>
-      <span>{deviceSize(partition.size)}</span>
+      <SplitItem>{deviceSize(partition.size)}</SplitItem>
+      <SplitItem>{partition.description}</SplitItem>
+      {label && (
+        <SplitItem>
+          <Label isCompact>{label}</Label>
```

Comment:
As discussed at G1 room, we should consider to use [outlined labels](https://www.patternfly.org/components/label#outlined-labels) instead. Reason: avoid confusion among widgets. At the time of writing, storage selectors and gray labels are quite similar in appearance.